scripts/build/p-llvm.inc

```diff
@@ -216,9 +229,11 @@ build_llvm() {
   CONFIG+=(
      "-DLLVM_INCLUDE_EXAMPLES=OFF"
 #     "-DLLVM_INCLUDE_TESTS=OFF"
```

Comment:
Remove this?

Reply:
I would rather like to add this eventually, but in this case not all LLVM versions build. Just as a small reminder.